docs/geedocs/answer/2987876.html

```diff
@@ -58,5 +75,5 @@
 <li>Tornado</li>
 <li>Xerces-c</li>
 </ul>
-<div class="footer"><p class="BackToTop"><a href="#top_of_file">Back to top</a> <hr /></p> <p class="copyright">&copy;2015 Google</p></div>
+<div class="footer"><p class="BackToTop"><a href="#top_of_file">Back to top</a> <hr /></p> <p class="copyright">&copy;2015 Google; &copy;2017 Open GEE Contributors</p></div>
```

Comment:
Should we change Google's copyright date from 2015 to 2017 since we did so at the top?

Reply:
In this case, I assumed the footer refers to the date on the page, which is why I didn't change it. At the top I assumed it applies to everything in the repo.